content/.contentrc

```diff
@@ -1 +1 @@
-{"length":2,"tagsDict":{"a":["62e2c530-32d7-4b17-835c-b02f8a137a4a","75b42dd5-6cd5-419e-bb7b-4fa3841dd5d0"],"b":["62e2c530-32d7-4b17-835c-b02f8a137a4a"],"c":["75b42dd5-6cd5-419e-bb7b-4fa3841dd5d0"],"d":["75b42dd5-6cd5-419e-bb7b-4fa3841dd5d0"]}}
```

Comment:
NOTE: the length is now wrong because the update contentrc script will only run on merge to master: [travis line#15](https://github.com/mottaquikarim/spectacles/pull/34/files#diff-354f30a63fb0907d4ad57269548329e3R15)